src/CLI/OptionsEvaluator.cs

```diff
@@ -22,7 +22,11 @@ public static IOptions ProcessInputs(Options rawInputs, IProcessHelper processHe
             }
             else if (!string.IsNullOrEmpty(processName))
             {
-                string p = Path.GetFileNameWithoutExtension(processName);
+                string p = Path.GetFileName(processName);
```

Comment:
How about breaking the new logic out into a function?

Reply:
Done